bool use_high_resolution_color
---
bool success